### ISSUE TITLE
Fatal error when running on Ubuntu 16.04.2 (64-bit) and Java 9

### ISSUE THREAD

**Issue by jmcastagnetto** [syvaidya/openstego#15]
Installed OpenStego using the deb from the [0.7.1](https://github.com/syvaidya/openstego/releases/download/openstego-0.7.1/openstego_0.7.1-1_amd64.deb) release in a desktop running an Ubuntu 16.04.2 LTS (64-bit). It did not run when using the GUI (.desktop file), so I tried using the CLI, and it gave the following error:

```
$ java -jar /usr/share/openstego/lib/openstego.jar 
net.sourceforge.openstego.OpenStegoException: java.lang.NullPointerException
	at net.sourceforge.openstego.util.PluginManager.loadPlugins(PluginManager.java:65)
	at net.sourceforge.openstego.OpenStego.main(OpenStego.java:427)
Caused by: java.lang.NullPointerException
	at net.sourceforge.openstego.util.CommonUtil.getStreamBytes(CommonUtil.java:54)
	at net.sourceforge.openstego.util.PluginManager.loadPlugins(PluginManager.java:51)
	... 1 more
```

As a reference:

```
$ java  -version
java version "9"
Java(TM) SE Runtime Environment (build 9+175)
Java HotSpot(TM) 64-Bit Server VM (build 9+175, mixed mode)
```

```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.2 LTS
Release:	16.04
Codename:	xenial
```

FYI - It does work when using **Java 8** on the same machine.

**Comment by syvaidya** [comment]
Java 9 is still not officially released, and will be released in September only. Current builds might have bugs. Please wait for official release and test with the same. If issue still persists, I'll take a look into it.

**Comment by madushan1000** [comment]
Hi,
I'm facing this issue now on Windows 10 64bit with Java 9 official release. This should be re opened.

**Comment by syvaidya** [comment]
Fixed with release 0.7.2
